azure-communication-ui/demo-app/src/calling-test/java/com/azure/android/communication/ui/callingcompositedemoapp/CallingCompositeScreenOrientationTest.kt

```diff
@@ -39,8 +39,12 @@ class CallingCompositeScreenOrientationTest : BaseUiTest() {
     }
 
     @Test
-    fun testSetOrientationPortrait() {
-        testSetupOrientation("PORTRAIT")
+    fun testSetSetUpOrientationPortrait() {
```

Comment:
Do you have any thought on how can we make 1 to N call UI test automated in local and in build pipeline?